pkg/service/kubeconfig_settings.go

```diff
@@ -54,6 +55,26 @@ func (kss *kubeconfigSettingService) Patch(ctx context.Context, ks *sentry.Kubec
 	if err != nil {
 		accId = uuid.Nil
 	}
+	const maxSeconds = 30 * 24 * 60 * 60
+	const minSeconds = 10 * 60
+
+	minTimeDuration := time.Second * time.Duration(minSeconds) // min. 10 mins
+	maxTimeDuration := time.Second * time.Duration(maxSeconds) // max. 30 days
+
+	validityDuration := time.Second * time.Duration(ks.ValiditySeconds)
+	if validityDuration < minTimeDuration || validityDuration > maxTimeDuration {
```

Comment:
Regarding sa validity, the changes are because there was no validation earlier for kubeconfig and secondly this was only dealing with hours that was restricting user choices

Reply:
reverted validation